src/express.ts

```diff
@@ -0,0 +1,16 @@
+import * as express from "express";
+import { IResponse } from "./responses";
+
+/**
+ * Convenience method that transforms a function (handler),
+ * which takes an express.Request as input and returns an IResponse,
+ * into an express controller.
+ */
+export function toExpressHandler<T>(
+  handler: (req: express.Request) => Promise<IResponse<T>>
+): <P>(req: express.Request, res: express.Response, object?: P) => void {
```

Comment:
shouldn't the return type be `express.RequestHandler` ?

Reply:
yes we get rid of the object param, see below

Reply:
good